src/plugin/morphologyHDF5.cpp

```diff
@@ -283,14 +301,15 @@ int MorphologyHDF5::_readSections()
         std::vector<std::vector<int>> vec;
         vec.resize(dims[0] - 1);
         dataset.read(vec);
-        bool skipFirst = true;
         int firstSectionOffset = vec[1][0];
+        sections.reserve(sections.size() + vec.size() - 1);
+        bool skipFirst = true;
         for (auto p : vec) {
```

Comment:
I wonder if `p` is copied from each `vec[i]` and if it has any effect on the performance.

Reply:
I'll change that. This could also receive a `const`.